syft/frameworks/torch/hook/hook.py

```diff
@@ -757,7 +766,7 @@ def new___init__(cls, *args, owner=None, id=None, register=True, **kwargs):
                 hook_self=hook_self,
                 cls=cls,
                 id=id,
-                torch_tensor=torch_tensor,
+                is_tensor=torch_tensor,
```

Comment:
Just tensor instead of is_tensor (is tensor makes me think its a boolean)

Reply:
It is a Boolean! 😱

Reply:
Can we change the name of the parameter in the surronding function then?